docker/systemd-test/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM eniocarboni/docker-ubuntu-systemd:22.04
+
+RUN apt-get update && \
+    apt-get -y --no-install-recommends install \
+    curl && \
+    rm /usr/sbin/policy-rc.d
```

Comment:
Nit: add `&& rm -rf /var/lib/apt/lists/*` for smaller container size

Reply:
The installer calls `apt-get install` internally. That would break if we remove this directory.